lib/incito-browser/incito.ts

```diff
@@ -283,7 +283,9 @@ function renderView(view, canLazyload) {
                     .forEach((style) => {
                         const {0: key, 1: value} = style.trim().split(':');
 
-                        styles[key] = value;
+                        if (key && key.length && value && value.length) {
+                            styles[key] = value.trim();
```

Comment:
`value` could be an empty string after the `trim()`, consider trimming before the above `if `clause